src/components/Menus/VerticalMenu/useVerticalTabs.tsx

```diff
@@ -17,11 +17,7 @@ export const useVerticalTabs = () => {
   const { t } = useTranslation();
 
   const handleClickTab = (tab: string) => () => {
-    const searchParams = new URLSearchParams(window.location.search);
-    let path = searchParams.toString();
-    path = path.startsWith('?') ? path.substring(1) : path;
-
-    router.push(`/${tab}?${path}`);
+    router.push(`/${tab}`);
```

Comment:
can you keep everything and just remove the "toToken" from the path  pls @oktapodia